route_info_android.go

```diff
@@ -0,0 +1,34 @@
+package sockaddr
```

Comment:
I believe you need a build tag here.

Reply:
I believe having `_android` suffix in the file name is sufficient enough, but added `// +build android` to be more explicit

Reply:
Ah you may be right, I don't deal with platform specific code much and forgot about that. But sure, being more explicit doesn't hurt!

Reply:
Actually, please remove it as there are already files in the package using the file name suffix pattern, so we might as well follow package convention.

Reply:
I removed it.